packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -292,11 +292,14 @@ class Scaffold extends StatefulWidget {
     this.drawer,
     this.bottomNavigationBar,
     this.backgroundColor,
-    this.resizeToAvoidBottomPadding: true
-  }) : super(key: key);
+    this.resizeToAvoidBottomPadding: true,
+    this.primary: true,
+  }) : super(key: key) {
+    assert(primary != null);
```

Comment:
please inline this assert and make the constructor const while you're here.

Inlining asserts and making Widget constructors `const` where possible is the new "add a trailing comma" of code reviews.